docs/developing.md

```diff
@@ -7,27 +7,39 @@ Hypothesis is comprised of several components:
    the client uses the public service at [hypothes.is](https://hypothes.is).
  - A browser extension which can add the client to web pages.
 
-In order to develop the client and test out changes you make locally, you will
-need to use either the [browser
-extension](https://github.com/hypothesis/browser-extension) or setup a [a local
-install of the Hypothesis
-service](http://h.readthedocs.io/en/latest/developing/) to host the client.
+## Prerequisites
 
-If you are only interested in making changes to the client, developing using
-the browser extension is the easiest method.
+You will need:
 
-## Prerequisites and Installation
+* [git](https://git-scm.com/)
+* [Node.js](https://nodejs.org/en/) v6+
 
-To develop the client, you will need [Node.js v6](https://nodejs.org/en/) or
-later and [gulp](https://github.com/gulpjs/gulp-cli). To install dependencies
-run:
+## Installation
+
+To install the client for development:
 
 ```sh
-npm install -g gulp-cli
+git clone 'https://github.com/hypothesis/client.git'
+cd client
+sudo npm install -g gulp-cli
```

Comment:
sudo is typically not required on OS X. This is why most guides tend to omit it. Linux users who are already familiar with npm will know they need to do this.